vslib/src/sai_vs_hostintf.cpp

```diff
@@ -671,7 +671,7 @@ void hostif_info_t::process_packet_for_fdb_event(
 
     if (fi.fdb_entry.bv_id == SAI_NULL_OBJECT_ID)
     {
-        SWSS_LOG_WARN("skipping mac learn for %s, since BV_ID was not found for mac",
+        SWSS_LOG_DEBUG("skipping mac learn for %s, since BV_ID was not found for mac",
```

Comment:
In what case we hit this?  This needs to be understood, and I don't think we should filter this out.

Reply:
We hit this during MAC learning. More precisely, when we have physical port and VLAN but fail to find bvid and bridge port ID. Also note that this happens for ports created during system initialization. For the ports created during breakout, we hit the following message:
```
 SWSS_LOG_WARN("failed to get port vlan id from port %s",
                        sai_serialize_object_id(portid).c_str());
```
I dont know how can this happen, because when we create port, we put it in default VLAN. When we try to get VLAN ID attribute of a port, it should always succeed. 

For now let me keep this log as warning and understand better.